terraform.advisories.yaml

```diff
@@ -24,3 +24,10 @@ advisories:
         type: fixed
         data:
           fixed-version: 1.5.5-r6
+
+  - id: GHSA-m425-mq94-257g
```

Comment:
Don't we have the normalized CVE ID ? So we can use that one instead, and set this one as the alias.

Reply:
looks like they dont have direct connection

Reply:
Okay. It can happen sometimes.